src/main/java/com/google/sps/data/Attendee.java

```diff
@@ -0,0 +1,79 @@
+package com.google.sps.data;
+
+import java.util.Date;
+import com.google.appengine.api.datastore.Entity;
+
+/** Class that represents a user in the session as an attendee. */
+public class Attendee implements AttendeeInterface{
+
+  // The id of the session this attendee is in.
+  private final String sessionId;
+
+  // The screen name of this attendee.
+  private final String screenName;
+
+  // Time the attendee last polled the server.
+  private Date timeLastPolled;
+  
+  /** Initializes an Attendee object
+   * @param {String} sessionId - the session the attendee is attached to.
+   * @param {String} screenName - the screen name of the attendee. 
+   * @param {Date} timeLastPolled - the time attendee was last polled.
+   */
+  public Attendee (String sessionId, String screenName, Date timeLastPolled) {
+    this.sessionId = sessionId;
+    this.screenName = screenName;
+    this.timeLastPolled = timeLastPolled;
+  }
+
+  public String getSessionId() {
+    return sessionId;
+  }
+
+  public String getScreenName() {
+    return screenName;
+  }
+
+  public Date getTimeLastPolled() {
+    return timeLastPolled;
+  }
+
+  /** Compares two Attendee objects to see if the values are the same. */
+  public boolean isEqual(Attendee x) {
+    boolean isEqual = false;
+      if(this.sessionId == x.sessionId) {
+        if(this.screenName == x.screenName) {
+          if(this.timeLastPolled == x.timeLastPolled) {
+            isEqual = true;
+          } 
+        }
+      }
+    return isEqual;
+  }
```

Comment:
I think the parameter for this function should be Object obj
You first want to make sure that the obj isn't null, if it is, then there is no equality.
Next, you want to make sure that the classes are the same at runtime, getClass() == obj.getClass(), 
next you want to downcast the object to match an Attendee -> Attendee otherAttendee = (Attendee) obj

Then you can compare the fields -> return sessionId.equals(otherAttendee.sessionID) && so on

Reply:
`public boolean isEqual(Object obj) {
  if(obj == null) return false;
  if(obj.getClass() != getClass()) return false;
  Attendee comparingToAttendee = (Attendee) obj;
  return this.sessionID(comparingToAttendee.sessionID) && the rest
}
`